Sources/SpeziCLAID/CLAIDRegistry.swift

```diff
@@ -0,0 +1,109 @@
+import Foundation
+import SwiftPython
+import PythonKit
+
+import CLAID
+import SwiftProtobuf
+
+public class CLAIDRegistry  {
+    
+    // Stores details about Python modules added to CLAID.
+    // We use this information to let the PyCLAIDLoader
+    // import the corresponding PythonFiles and add the
+    // CLAID Python Modules to the Python Module factory.
+    public var pythonModuleDetails: [PythonModuleDetails] = []
+    // Holds a list of CLAID ModuleConfigs, which are normally extracted from
+    // the JSON config file when loaded. During startup we merge these configs
+    // with the config object loaded from the JSON file.
+    public var moduleConfigurations: [Claidservice_ModuleConfig] = []
+    public var preloadedCLAIDModules: [String:Module] = [:]
+    
+    private func makeModuleConfig(
+        moduleId: String,
+        moduleType: String,
+        properties: [String: String],
+        inputChannels: [String: String] = [:],
+        outputChannels: [String:String] = [:]
```

Comment:
I am generally wondering if we can make these identifiers type-safe in the long run. For other parts of Spezi, we have introduced identifier types (e.g. Storage, Access Guard, HealthKit bulk upload, ...), I think these mechanisms would ensure that different elements are safer down the line (with always having an escape hatch to make them string-based).